### PR TITLE
Minor grammar and spelling fixes.

### PR DESCRIPTION
**Pull request by johnnymatthews**

### CODE REVIEW

File: src/_blog/libp2p-hole-punching.md [hunk 1]
```diff
@@ -37,130 +36,128 @@ The simplest security measure all major consumer firewalls enforce to protect th
...
 
 # Hole Punching
 
 Hole punching is one mechanism to overcome the problem described above. Let's see how it works.
 
-Imagine that we have some mysterious mechanism to synchronize *A* and *B*. Mysterious as synchronizing clocks is hard, &#x2026; really hard. For those that don't believe me, I recommend reading Lamport's [_Time, Clocks, and the Ordering of Events in a Distributed System_](https://web.archive.org/web/20211101121036/https://amturing.acm.org/p558-lamport.pdf) and Mills' [_A Brief History of NTP Time: Memoirs of an Internet Timekeeper_](https://web.archive.org/web/20220120075303/https://www.eecis.udel.edu/~mills/database/papers/history.pdf).
-
-Anyways, back to assuming the existence of a mysterious synchronization mechanism. Such mechanism would allow *A* and *B* to dial each other "at the same time".
+Imagine that we have some mysterious mechanism to synchronize *A* and *B*. Such a mechanism would allow *A* and *B* to dial each other "at the same time".
```
Comment (mxinden) [inline src/_blog/libp2p-hole-punching.md:62]: Why remove these further reading suggestions @johnnymatthews?

**Review by mxinden — COMMENTED** [review]
Thanks for the help. Very much appreciated! :pray:

Small follow-up below:

File: src/_blog/libp2p-hole-punching.md (hunk 1 above)
Comment (johnnymatthews) [reply src/_blog/libp2p-hole-punching.md:62]: Ah, my bad. Didn't meant to take it out. But I'm a bit confused why it's in here. What does this _mysterious clock synchronisation_ have to do with the rest of this post? I might be missing something, but this felt very out of place, as was only put in here to mention these two articles from Lamport and Mills.

Comment (mxinden) [reply src/_blog/libp2p-hole-punching.md:62]: That makes sense. I think these are great papers to read on time synchronization. But as you say, time synchronization only plays a minor role in this blog post, thus referencing these is too much.

(Also don't want to come across as just _name dropping_.)

Thanks @johnnymatthews.

**Review by mxinden — APPROVED** [review]
Again, thanks for the help!

**Pull request merged**